gn3/db/rdf/wiki.py

```diff
@@ -180,3 +187,112 @@ def get_comment_history(
     # See note above in the doc-string
     results["data"] = sorted(data, key=lambda d: d["version"], reverse=True)
     return results
+
+
+def get_next_comment_version(
```

Comment:
what should be the source of truth for this function? There's a similar function for this [here](https://github.com/jnduli/genenetwork3/blob/52cd294c2f1d06dddbd6ff613b11f8bc43066038/gn3/db/wiki.py#L51) and I'd rather we stick to one. This prevents drifts when problems occur.